src/Views/SilenceForm/Views.elm

```diff
@@ -0,0 +1,106 @@
+module Views.SilenceForm.Views exposing (editForm, newForm)
+
+import Html exposing (Html, div, text, fieldset, legend, label, span, a)
+import Html.Attributes exposing (class, href)
+import Html.Events exposing (onClick)
+import Silences.Types exposing (Silence)
+import Types exposing (Msg(MsgForSilenceList, PreviewSilence, MsgForSilenceForm), Model)
```

Comment:
I don't think we should depend on the top level `Model` type here. `editForm` and `newForm` only require the silence type.